arcgis-ios-sdk-samples/Content Display Logic/Storyboard/Main.storyboard

```diff
@@ -78,13 +78,33 @@
                                     <action selector="closeAction" destination="HRz-Eg-7ly" eventType="touchUpInside" id="JSz-9v-XS9"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="erh-VF-k0q">
+                                <rect key="frame" x="52" y="491" width="155" height="30"/>
+                                <state key="normal" title="Help"/>
```

Comment:
I was surprised to see that this button was titled Help, but it took me to a community forum. I would expect it to take me to a help page specifically for the sample view, which would then have links to the guide, the reference doc, the community, etc. That would also mean we would only need one button, not two. I was also confused by the title of the other button, for I expected it to show me more info about the sample app, not the SDK itself.